app/controllers/sessions_controller.rb

```diff
@@ -42,5 +42,19 @@ def find_user
 
   def new_spire
     (User.pluck(:spire).map(&:to_i).max + 1).to_s.rjust(8, '0') + '@umass.edu'
+
   end
 end
+
+
```

Comment:
I think one newline is good enough here

Reply:
Fun fact: the newline at EOF isn't even a real carriage return. It's a hidden character.